sharding-core/sharding-core-execute/src/main/java/org/apache/shardingsphere/core/execute/sql/execute/result/QueryResultUtil.java

```diff
@@ -53,8 +53,6 @@ public static Object getValue(final ResultSet resultSet, final int columnIndex)
     public static Object getValueByColumnType(final ResultSet resultSet, final int columnIndex) throws SQLException {
         ResultSetMetaData metaData = resultSet.getMetaData();
         switch (metaData.getColumnType(columnIndex)) {
-            case Types.BIT:
```

Comment:
Please refer to #2460, the BIT was added to resolve a sharding-proxy issue. This change will make sharding-proxy return wrong value.

Reply:
> return field.isSingleBit() ? Boolean.valueOf(getBoolean(columnIndex)) : getBytes(columnIndex);

getObject() can return correct result

Reply:
Have you tested? Please show your test cases for sharding-proxy.